src/js/components/LogItem/utils/createCommandHeaderDocLink.tsx

```diff
@@ -0,0 +1,15 @@
+const baseUrl = 'https://github.com/jooby-dev/jooby-docs/blob/main/docs/message.md#';
+
+
+export const createCommandHeaderDocLink = (headerLength: number) => {
```

Comment:
let's discuss default export here and in other places

Reply:
discussed

changed to export by default